packages/run-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -232,29 +361,43 @@ export const makeVaultManager = (
       want: { RUN: null },
     });
 
-    const startTimeStamp = await E(timerService).getCurrentTimestamp();
+    // eslint-disable-next-line no-plusplus
+    const vaultId = String(vaultCounter++);
```

Comment:
If the style guide asks us to avoid ++, there  isn't a good reason to violate that rule here. Doing what the style guide wants is at least as concise and understandable as the work to disable it.

```suggestion
    vaultCounter += 1;
    const vaultId = String(vaultCounter);
```

Reply:
Agree completely about not violating style rules. I had a PR to remove this as a rule but it's closed and this is the right way.